CTestConfig.cmake

```diff
@@ -34,7 +34,7 @@ else ()
   if (CDASH_LOCAL)
     set (CTEST_DROP_LOCATION "/submit.php?project=HDF5112")
   else ()
-    set (CTEST_DROP_LOCATION "/submit.php?project=HDF5")
+    set (CTEST_DROP_LOCATION "/submit.php?project=HDF5112")
```

Comment:
Test scripts on HPC machines sending results to my.cdash.org have to override this (HDF5112 to HDF5), otherwise the results just disappear.  It's annoying, but we can live with it.